roles/lokole/templates/lokole.conf.j2

```diff
@@ -0,0 +1,27 @@
+# Root directory goes to Lokole web server
+
```

Comment:
```suggestion

RedirectMatch {{ lokole_url }} {{ lokole_url }}/

```

Reply:
How would this fix the issue?

Reply:
> How would this fix the issue?

Magic!  In any case: http://box/lokole now redirect to http://box/lokole/ and the issue is resolved.  As merged into IIAB 6.7/master and tested on Ubuntu 18.04 and Raspbian Lite 2018-10-09.